src/components/WalletModal/index.tsx

```diff
@@ -211,61 +219,85 @@ export default function WalletModal({
         })
   }
 
+  const handleWalletActivation = (item: any) => {
+    try {
+      connect(item)
+      if (item.id() === 'argentX') {
+        localStorage.setItem('auto-injected-wallet', 'argentX')
```

Comment:
please replace the raw wallet name strings with values from constants. we will be more protected from typos in this case. I think you can add new fields with the short names [here](https://github.com/jediswaplabs/jediswap-interface/blob/main/src/constants/index.ts#L179), I would call it `id`.

It will be something like:

```localStorage.setItem('auto-injected-wallet', SUPPORTED_WALLETS['ArgentX'].id)```

but the details are up to you.